package.json

```diff
@@ -74,7 +74,7 @@
   "author": "Juan Cruz Viotti <juan@balena.io>",
   "license": "Apache-2.0",
   "engines": {
-    "node": ">=8.0"
+    "node": ">=10.0"
```

Comment:
@thgreasi, as discussed in Google chat (balena-io/product), would it be possible to avoid this Node.js bump in this PR, and have this PR's change-type be minor or patch rather than major? And then we merge this PR in CLI v11, and I can add the "balenaCloud backend support/deprecation policy" to the CLI.

Reply:
@pdcastro you mean, even though the SDK's package.json declares that the minimum supported node version is v10?
I haven't tried it, but I think that while doing a fresh npm install on node 8, npm will complain as soon as it sees that the sdk needs node 10.